metadata/src/main/java/org/apache/kafka/controller/QuorumController.java

```diff
@@ -1481,8 +1498,7 @@ private void resetToEmptyState() {
         snapshotGeneratorManager.cancel();
         snapshotRegistry.reset();
 
-        newBytesSinceLastSnapshot = 0;
```

Comment:
it seems to me like resetToEmptyState should still set this to 0.

Reply:
Yes. We are still doing this when we call `updateLastCommittedState(-1, -1, -1, 0)`. Notice the subtle `0` for the last parameter.

Reply:
oh, sorry. LGTM then.